src/man/firejail.1.in

```diff
@@ -1729,6 +1736,10 @@ See --keep-config-pulse.
 Disable blacklist for this directory or file.
 .br
 
+Note that this only applies to the path as written when it was blacklisted,
+including unexpanded variables and wildcards.
```

Comment:
Not sure this is correct.

If somebody wants to test, here are the test cases to construct the test matrix:

- "simple" paths
- macros w/o subpaths (e.g. `${PICTURES}`, `${DOCUMENTS}`, ...)
- macros w/ subpaths (e.g. `${HOME}`)
- macros with multiple maths (`${PATH}` - the handling of this macro is more special than the other ones)
- wildcards (`*`, ...) in only (no)blacklist and both.

Reply:
You're right, the behavior is different for wildcards, in that wildcard blacklists *can* be partially unblacklisted by `--noblacklist`ing individual expansions instead of only the whole wildcard.

As for variables, I only tested it for `${PATH}` - didn't realize this was a special case, but in retrospect it makes sense that it is :facepalm:

Reply:
All right, I tested the `${PATH}` situation again with another executable I created myself just to be sure that it isn't specific to `nc` for some reason, and yes, `${PATH}` really does behave differently than all other expansions (variables & wildcards).

I'm inclined to say that this is probably a bug and hence shouldn't be documented but rather get an issue? I'll create one.

Reply:
The reason `${PATH}` is special is because it can expand into multiple paths while `${HOME}` and the like are only substituted in the path.

Reply:
Issue: https://github.com/netblue30/firejail/issues/6360

Reply:
I've changed the paragraph so it accounts for #6360 and links to it, just in case it's decided that #6360 either takes too long to fix or is not considered a bug.